prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -253,12 +253,13 @@ public override async Task<bool> ProcessRule(Enrollee enrollee)
         {
             var paperEnrollees = await _enrolleePaperSubmissionService.GetPotentialPaperEnrolleeReturneesAsync(enrollee.DateOfBirth);
             var potentialPaperEnrolleeGpid = await _enrolleePaperSubmissionService.GetLinkedGpidAsync(enrollee.Id);
+            var isEnrolleeLinked = await _enrolleePaperSubmissionService.IsEnrolleeLinkedAsync(enrollee.Id);
```

Comment:
This is only used once in the method you can inline it into the if-statement.

Reply:
inlined